t/full-stack.t

```diff
@@ -525,5 +527,18 @@ END {
     kill_driver;
     turn_down_stack;
     session->clean;
+    # prevent double execution
+    return if $? == 0;
     $? = 0;
+    find(
+        sub {
+            if (-f and /\.txt$/) {
+                print("$_:\n");
```

Comment:
i think using  `print "$_ contents: ".path($_)->slurp;` is a bit more elegant. https://mojolicious.org/perldoc/Mojo/File#path

Reply:
yes, works. thx